torch/_dynamo/utils.py

```diff
@@ -3106,3 +3107,43 @@ def get_user_object_from_id(obj_id):
 def store_user_object_weakref(obj):
     obj_id = id(obj)
     user_obj_id_to_weakref[obj_id] = weakref.ref(obj)
+
+
+class CompileTimeInstructionCounter:
+    _counter: int = 0
+    _id: int = -1
+
+    @classmethod
+    def start(cls) -> None:
```

Comment:
Maybe annotate all these functions?

Reply:
what do you mean

Reply:
Oh - I'm being silly here. @classmethod cls doesn't need annotation - but below the
```
def __init__(self):
```
should be:
```
def __init__(self) -> None:
```
(and enter and exit need type annotations if you don't switch to a contextmanager.